src/lib/wifiScanner.ts

```diff
@@ -127,26 +127,55 @@ export function parseNetshOutput(output: string): WifiNetwork {
 
   const lines = output.split("\n");
 
+  let SSID = "";
+  let BSSID = "";
+  let Signal = "";
+  let Channel = "";
+  let Authentication = "";
+  let TR = "";
+  let phyMode = "";
+
+  lines.forEach((line) => {
+    const trimmedLine = line.trim();
+
+    if (trimmedLine.startsWith("Name")) {
+      SSID = "SSID";
+      BSSID = "BSSID";
+      Signal = "Signal";
+      Channel = "Channel";
+      Authentication = "Authentication";
+      TR = "Transmit rate (Mbps)";
+      phyMode = "Radio Type";
+    } else if (trimmedLine.startsWith("Nome")) {
+      SSID = "SSID";
+      BSSID = "BSSID";
+      Signal = "Segnale";
+      Channel = "Canale";
+      Authentication = "Autenticazione";
+      TR = "Velocità trasmissione (Mbps)";
+      phyMode = "Tipo frequenza radio";
+    }
+  });
```

Comment:
Generally good, but please instead move this into `getFieldIdentifiers(output)`, check the language there and then return an object such as `{bssid: "BSSID", signalStrength: "Segnale", ...}`